create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/utils/check-requirements.py

```diff
@@ -0,0 +1,61 @@
+import subprocess
+import os
+import click
+from functools import partial
+from pathlib import Path
+
+if __name__ == '__main__':
+    path = Path('../../requirements/')
+
+    for file in os.listdir(path):
+        with open("../../requirements/" + file, "r") as text:
+            old_dependencies = text.read().split('\n')
+            new_dependencies = []
+
+            for line in old_dependencies:
```

Comment:
For me code is complicated and some time i don't know what is going on (as example `old_line.find('jinja2')`). I think, it's because u wanna coverage all cases. 

What do I suggest?

*first step* - get a new version of packages
```python
import subprocess

...
fresh_version = subprocess\
    .check_output("pip list -o --format=freeze", shell=True)\
    .decode("utf-8")\
    .split("\n")

packages = dict(package.split('==') for package in fresh_version if package)
```
result
```
print(packages)

>>> {'Markdown': '2.6.10',
>>> 'Rx': '1.6.0',
>>> ...
```
*second step* - rewrite versions for the each file

```python
with req_file.open() as f:
    text = f.read()
    for name, version packages.items():
        text = re.sub(f'{name}==[\d .]*', f'{name}=={version}', text) 
    # write to the file
    f.seek(0)
    f.write(text)
    f.truncate()
```

what do u think?

Reply:
Yeah, the problem has changed, but the methods remain the same. Instead of rewriting the code after the first review, I tried to change it by overloading the solution. Classic. :smiley:
Your propose looks more elegant.:fire: I like it. 
Thanks for this review. I learned a lot of new things. :thumbsup: